### PR TITLE
Dg/rum 588/prep repo for alpha

### PR DESCRIPTION
**Pull request by raygun-harrison**
Updates names of files and demo. Removing SundayEmpire references and updating version value.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/sundayempire/raygun4reactnative.git"
+    "url": "git+https://github.com/raygun4reactnative.git"
```
Comment (mduncan26) [inline package.json:22]: This will need the organisations name within the URL. 

"git+https://github.com/MindscapeHQ/raygun4reactnative.git"

**Review by mduncan26 — CHANGES_REQUESTED** [review]
One thing to touch up.

File: package.json (hunk 1 above)
Comment (mduncan26) [reply package.json:22]: Also I noticed the author.name being "name": "MindscapeHQ", which we should change to "name": "Raygun",

Comment (RaygunDaniel) [reply package.json:22]: Sweet I'll get onto it now

**Review by mduncan26 — APPROVED** [review]
LGTM

**Pull request merged**